service.go

```diff
@@ -68,3 +76,14 @@ func dumpRepositoriesRecursive(
 		}
 	}
 }
+
+// moduleEnabledInEnv returns true if the provided `identifier` is included in
+// the `ADDITIONAL_MODULES` environment variable, otherwise false.
+func moduleEnabledInEnv(identifier string) bool {
+	for _, i := range strings.Split(os.Getenv("ADDITIONAL_MODULES"), ",") {
+		if identifier == i {
+			return true
+		}
+	}
+	return false
+}
```

Comment:
This function is going to be called several times, maybe it would make more sense to have a list or a dictionary (to have faster access at comparison time) with all the modules the user enabled.

Feel free to shut me down in case you think this is a premature optimization (it kinda feels like that even to me).

Reply:
I actually thought the same during the implementation, but in the end I think this is one of the most compact ways to represent it in go. Another option would be to parse the env var and pass the result around during the comparison with the module identifier.

I would like to keep it this way for now, since the performance costs should be really low. :)

Reply:
:+1: